modules/tests/jvm/src/main/scala/lucuma/catalog/GaiaQueryPMApp.scala

```diff
@@ -48,7 +50,9 @@ trait GaiaQueryPMSample {
     val query = CatalogSearch.gaiaSearchUri(
       TimeRangeQueryByADQL(
         tracking,
-        Bounded(Instant.EPOCH, Instant.EPOCH.plusSeconds(365 * 24 * 60 * 60 * 60), 0),
+        Interval
+          .closed(Instant.EPOCH, Instant.EPOCH.plusSeconds(365 * 24 * 60 * 60 * 60))
+          .asInstanceOf[Bounded[Instant]],
```

Comment:
A change in spire for scala 3 requires the `asInstanceOf`?

Reply:
Sorry, missed this notification. Let me take a look.

Reply:
It is standard practice for `case class` constructors to be private. But why can't you use the `apply` method?
https://github.com/typelevel/spire/blob/1c8765edcb056a102e3e537b5eb0839141142369/core/src/main/scala/spire/math/Interval.scala#L817

Reply:
I looked at this, and it seems that somehow making the constructor `private` also makes the `apply` method `private` as well. Will investigate further ...

Reply:
Yeah, this seems to be a Scala 3 thing. The following code compiles:
```scala
//> using option "-source:3.0-migration"

case class Foo private (bar: Int)

@main def main = Foo(0)
```

But if you remove the `-source` scalac flag then it gets you:
```
[error] ./bug.scala:4:18: method apply cannot be accessed as a member of Foo.type from module class bug$package$.
[error] @main def main = Foo(0)
[error]                  ^^^
```

Reply:
Here's the dotty PR, from a while back ... I guess this new semantic is pretty baked in.
- https://github.com/lampepfl/dotty/pull/5908

Reply:
Does this mean we could publish spire witth the migration flag to make it work correctly?

Reply:
Yes, if we published Spire with the migration flag then it would work. But enabling the migration flag will prevent the compilation of Scala 3-only features like macros and extensions I think, so it's not realistic.

Also, I don't think the new semantic is correct, not the old one. The old one was exposing an unsafe, non-validating constructor.